### PR TITLE
{debugger,devel}[GCCcore/8.2.0] GDB v8.3, texinfo v6.6 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GDB/GDB-8.3-GCCcore-8.2.0-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ('binutils', '2.31.1'),
+    ('texinfo', '6.6'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.1'),
+    ('expat', '2.2.6'),
+    ('Python', '3.7.2'),
```
Comment (smoors) [inline easybuild/easyconfigs/g/GDB/GDB-8.3-GCCcore-8.2.0-Python-3.7.2.eb]: can you use multi_deps instead?
`multi_deps = {'Python': ['3.7.2', '2.7.15']}`

Comment (Micket) [reply easybuild/easyconfigs/g/GDB/GDB-8.3-GCCcore-8.2.0-Python-3.7.2.eb]: No, it's not a python binding, it's built-in python, so it links the gdb binary to either libpythonX.so library, so there can only be one per PATH.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic25 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/09535869e14a9dd290ce8ae5151f74a7 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @Micket!

**Pull request merged**